### PR TITLE
add crnn int8(EN)

### PR DESCRIPTION
**Pull request by Charles-258**
This pr is to revise the Qoperator crnn int8 model, and provide another QDQ crnn int8 model.
The results of two int8 models can be found in models/text_recognition_crnn/README.md.

### CODE REVIEW

File: models/text_recognition_crnn/README.md [hunk 1]
```diff
@@ -8,8 +8,12 @@ Results of accuracy evaluation with [tools/eval](../../tools/eval) at different
 |--------------|------------|-----------|-----------|
 | CRNN_EN      | 81.66      | 74.33     | 52.78     |
 | CRNN_EN_FP16 | 82.01      | 74.93     | 52.34     |
+| CRNN_EN_INT8_QOperator |   81.75   |   75.33  | 52.43    |
+| CRNN_EN_INT8_QDQ |   79.84   |   74.47   |  51.04  |
```
[the remark this answers is not in the document]
Comment (fengyuentau) [reply models/text_recognition_crnn/README.md:12]: Same for the other model.

[the remark this answers is not in the document]
Comment (zihaomu) [reply models/text_recognition_crnn/README.md:12]: Hi @Charles-258, I agree with @fengyuentau. We should give users fewer choices and ensure that `opencv_zoo` has the best model.
I propose to remove `QDQ` version model and rename `INT8_QOperator` as the `INT8`.

**Comment by fengyuentau** [comment]
@Charles-258 friendly reminder.

**Review by fengyuentau — APPROVED** [review]
Need to have timestamps for both fp16 and int8 models and use lowercase for data type. Something like this:
```.diff
- text_recognition_CRNN_CH_INT8.onnx
+ text_recognition_CRNN_CH_2022oct_int8.onnx
```

Others look good to me. Please squash commits into one before merge.

**Review by fengyuentau — APPROVED** [review]
Thank you for your contribution! Everything looks good to me 👍

**Review by zihaomu — APPROVED** [review]
LGTM! 👍

**Pull request merged**